api/forum/models.py

```diff
@@ -1,3 +1,16 @@
 from django.db import models
+from accounts.models import Profile
 
-# Create your models here.
+
+class Post(models.Model):
+    parent = models.ForeignKey('self', related_name='comments', on_delete=models.CASCADE, null=True, blank=True, default=1)
```

Comment:
With just the `ForeignKey` I think you'll only be able to have one 'comment'. I think you'll need a many-to-many.